src/components/TutorialsOverview.vue

```diff
@@ -122,7 +122,8 @@ export default {
       padding-top: $nav-height-small;
     }
 
-    background: var(--color-tutorials-overview-background);
+    background: var(--color-tutorials-overview-fill-secondary,
```

Comment:
Is `--color-tutorials-overview-fill-secondary` like a more broad color to style a set of things at once?

Reply:
Yeah, there are two main background fill colors used in this page for the default style, so I needed to add a new var to support that, and I tried to make the variable names more semantic to better make that clear (background by itself implies that there is only one main background color, which isn't totally accurate).